cmd/get_logs.go

```diff
@@ -362,6 +365,17 @@ func (glc *getLogsCmd) getCloudName() string {
 	return ""
 }
 
+func (glc *getLogsCmd) getControlPanelNodes() []v1.Node {
+	var ControlPanelNodeList []v1.Node
```

Comment:
shorter camel-cased variable names are more idiomatic in my opinion

ex: `i` instead of `nodeIndex` or `node` instead of `controlPlaneNode`

Reply:
updated